fud2/src/main.rs

```diff
@@ -220,6 +304,13 @@ fn build_driver() -> Driver {
         verilog_noverify,
         firrtl_compile,
     );
+    bld.op(
+        "firrtl-with-primitives-noverify",
```

Comment:
I understand why, but it is annoying we have to duplicate these FIRRTL compilers for the "noverify" variants! Otherwise, the Verilog will work with only Icarus or Verilator but not both. The real solution here, of course, is to somehow make our extra checks work in Icarus so we don't need this distinction… but that is a problem for another day.